src/main/java/com/online/store/service/ProductServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.online.store.service;
+
+import com.online.store.dao.ProductDao;
+import com.online.store.lib.Inject;
+import com.online.store.lib.Service;
+import com.online.store.model.Product;
+import java.util.List;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+    @Inject
+    ProductDao productDao;
+
+    @Override
+    public Product create(Product product) {
+        return productDao.create(product);
+    }
+
+    @Override
+    public Product get(Long productId) {
+        return productDao.get(productId).orElseThrow();
```

Comment:
```suggestion
        return productDao.get(productId).get();
```
let's use `get()` for now

Reply:
done